### PR TITLE
Update for Angular 5.0

### PR DESCRIPTION
**Pull request by vikerman**

### CODE REVIEW

**Comment by Toxicable** [comment]
Could you do a rebase with master

File: README.md [hunk 1]
```diff
@@ -12,18 +12,37 @@ A minimal Angular starter for Universal JavaScript using TypeScript and Webpack
...
+### Installation
+* `npm install` or `yarn`
+
+### Development (Client-side only rendering)
+* run `npm run start` which will start `ng serve`
+
+### Production (also for testing SSR/Pre-rendering locally)
+**`npm run build:ssr && npm run serve:ssr`** - Compiles your application and spins up a Node Express to serve your Universal application on `http://localhost:4000`.
+
+**`npm run build:prerender && npm run serve:prerender`** - Compiles your application and prerenders your applications files, spinning up a demo http-server so you can view it on `http://127.0.0.1:8080`
```
Comment (Toxicable) [inline README.md]: maybe change the host to localhost like the other one

**Comment by Toxicable** [comment]
Could you add the `.vscode` directory to the .gitignore

File: README.md (hunk 1 above)
Comment (vikerman) [reply README.md]: Done

**Comment by CaerusKaru** [comment]
This also closes #481

**Pull request merged**